cmd/pop/cli/start.go

```diff
@@ -31,6 +31,7 @@ type PopConfig struct {
 	replInterval time.Duration
 	// Exported fields can be set by survey.Ask
 	Bootstrap    string `json:"bootstrap"`
+	MaxPPB       int    `json:"maxppb"`
```

Comment:
If the maxppb is saved in the JSON config file, can't this value be accessed when running the get command? I'm not a fan of setting it as a property of the `node` struct.

Reply:
But is it or will it be possible that the machine running the get command is not the same as the one who runs the Start? Like connecting to a remote pop node with websockets or RPC?

Because in this case the get() will not have access to the start config, that's why I thought when doing this PR

We are also already passing some startArgs in the `node` struct :
``` go
opts := node.Options{
		FilEndpoint:    startArgs.FilEndpoint,
		ReplInterval:   startArgs.replInterval,
		MaxPPB:         int64(startArgs.MaxPPB),
		BootstrapPeers: bAddrs,
		FilToken:       filToken,
		PrivKey:        privKey,
		Regions:        regions,
		Capacity:       capacity,
	}
```

So I was wondering, why not directly passing all the PopConfig to the `node` struct ? 
``` go
opts := node.Options{
		PopConfig:      startArgs, // FilEndpoint,  ReplInterval, MaxPPB will be there
		BootstrapPeers: bAddrs,
		FilToken:       filToken,
		PrivKey:        privKey,
		Regions:        regions,
		Capacity:       capacity,
	}
```

Reply:
Ok, let's keep it flat but then keep a reference to the node.Options on the `node` object so we can access any of the config from the node methods